tests/encryption_/test_base.py

```diff
@@ -0,0 +1,224 @@
+from datetime import datetime, time
+
+import pymongo
+from bson.binary import Binary
+from django.conf import settings
+from django.db import connections, models
+from django.test import TransactionTestCase, modify_settings, override_settings
+
+from django_mongodb_backend.fields import EncryptedFieldMixin
+
+from .models import (
+    Appointment,
+    Billing,
+    EncryptedNumbers,
+    Patient,
+    PatientPortalUser,
+    PatientRecord,
+)
+from .routers import TestEncryptedRouter
+
+
+class EncryptedDurationField(EncryptedFieldMixin, models.DurationField):
+    """
+    Unsupported by MongoDB when used with Queryable Encryption.
+    Included in tests until fix or wontfix.
```

Comment:
Can we make an INTPYTHON JIRA ticket for this? I don't see a solution anytime soon.